packages/import-ynab4/index.js

```diff
@@ -1,9 +1,7 @@
-#!/usr/bin/env node
-const fs = require('fs');
-
-const { init, shutdown } = require('@actual-app/api');
-
-const { importBuffer } = require('./importer');
+#!/usr/bin/env ts-node
```

Comment:
I think the `#!` can be removed along with `ts-node`, the `import-*` packages are deprecated as standalone tools (instead you can just use the buttons directly within the UI). Maybe do that in a separate PR though.

Reply:
Oh, I didn't know 😅 Yeah, maybe we should clean them up in another PR... `ts-node` was already used as peer dependency so if not really adding anything new. Do you want me to revert the changes anyway?

Reply:
Nah, it’s fine to keep them. ⏩ #878